src/main/java/Event.java

```diff
@@ -1,6 +1,6 @@
 public class Event extends Task{
-    private String fromDate;
-    private String toDate;
+    private final String fromDate;
+    private final String toDate;
```

Comment:
In line 2 and 3, regarding the coding standard, the final string should be in capital letters.